tools/qrgen/README.md

```diff
@@ -0,0 +1,17 @@
+# QR generation tools for Individual and Commercial
+
+[![License](https://img.shields.io/badge/licence-Apache%202.0-brightgreen.svg?style=flat)](LICENSE)
+
+## Summary
+
+This repository provides a tools to generate QR codes for ST devices.
+
+## Requirement
+
+## Prerequisites
```

Comment:
How about to add simple guide to install qrcode module?
```sh
pip3 install qrcode
```

Reply:
I've added info about need for installing qrcode Python package.

Since I'm using Ubuntu (without pip) I copied and modified the command from keygen tool for pip and added command for Ubuntu as well.